.hintrc

```diff
@@ -0,0 +1,18 @@
+{
+    "connector": {
+      "name": "local",
+      "options": {
+        "pattern": ["**", "!.git/**", "!node_modules/**"]
+      }
+    },
+    "extends": ["development"],
+    "formatters": ["stylish"],
+    "hints": [
+      "button-type",
+      "disown-opener",
+      "html-checker",
+      "meta-charset-utf-8",
+      "meta-viewport",
+      "no-inline-styles:error"
+    ]
+  }
```

Comment:
- [ ] Although you have the various lint files, you did not include the `.github/workflows` directory that contains the `linters.yml` file. This file has the configurations that will enable GitHub actions to run the file and check that your codes were written properly. Kindly include this directory in the root of this project. 👍 😎